Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,223 @@
+  index as key.
+- Now shift all pulses in all sets from the same region by ``-featurePos``
+  where ``featurePos`` is used from the same sweep and pulse index.
+
+.. _db_paplot_operation_order:
+
+Operation order
+^^^^^^^^^^^^^^^
+
+The operations on the data happens in the following fixed order:
+
+- Gather pulses
+- Apply pulse sort order
+- Failed pulse searching
+- Zeroing
+- Time alignment
+- Averaging
+
+PA plot settings
+^^^^^^^^^^^^^^^^
+
+The following figure shows the browser settings panel for the PA plot. It can
+be reached from the ``PA`` tab in the Databrowser. The next sections explain each control.
+
+.. _Figure Pulse Average Browser Settings:
+
+.. figure:: BrowserSettingsPanel-PA-plot.png
+   :align: center
+
+   Settings for the pulse average plot
+
+Pulse Averaging
+"""""""""""""""
+
+- ``enable``: Toggle the display of the PA image/trace plots.
+  Can be turned off before tweaking the settings and then reenabled to make
+  changing the settings quicker.
+
+Image settings
+""""""""""""""
+
+- ``Enable image plot``: Toggle the display of the :ref:`Figure Pulse Average Image plot`
+- ``Draw X zero line``: Draw a vertical line at ``X == 0`` when
+  time alignment is turned on. Helps with distinguishing true from false responses.
+- ``Popup Menu``: Color scheme for image plots
+- ``Sort order``: Sort order of the shown pulses. For ``Sweep`` the pulses are
+  first ordered by sweep number and then by pulse index. For ``Pulse`` it is
+  first pulse index and then sweep number. See also
+  :ref:`db_paplot_image_sortorder`.
+
+Trace settings
+""""""""""""""
+
+- ``Enable trace plot``: Toggle the display of the :ref:`Figure Pulse Average Trace plot`.
+- ``Vertical scale bar``: Size of the vertical scale bar in y-axis units
+
+Deconvolution
+"""""""""""""
+
+- ``Deconvolution``: Toggle the deconvolution pulse display, this is caculated
+  from the averaged pulses [#1]_. For the image plot, the average is the second
+  bottom-most row.
+- ``Tau [ms]``: Time constant [#2]_
+- ``Smoothing``: Smoothing parameter, use ``1`` to disable smoothing
+- ``Display [ms]``: Time range of the average pulse used for the deconvolution, use ``inf`` to use the full range
+
+General
+"""""""
+
+- ``Zero pulse``: Toggle Pulse Zeroing. This is done by Differentiation and Integration.
+- ``Time alignment``: Toggle time alignment of pulses from one column. See :ref:`db_paplot_timealignment` for an in-depth explanation.
+- ``Show average``: Toggle average pulse display. For the image plot, the average is the bottom-most row.
+- ``Use multiple graphs``: Toggle the display mode. When unchecked a single
+  graph is used for all traces and images, when checked a separate graph is
+  used for each PA set.
+- ``Show individual pulses``: Toggles the display of the single pulses. Turning
+  that off can increase the display update performance. The average and
+  deconvolution are still shown if enabled.
+
+Pulses
+""""""
+
```

Comment:
Select a subset of the pulses from a train of pulses (contained within a sweep).